posthog/api/capture.py

```diff
@@ -121,7 +121,7 @@ def _get_sent_at(data, request) -> Optional[datetime]:
         else:
             return None
 
-        if re.match(r"^[0-9]+$", sent_at):
+        if re.match(r"^[0-9.]+$", sent_at):
```

Comment:
This would also match something weird like `.4124.10943.32`, so maybe something more restrictive?
```suggestion
        if re.match(r"^\d+(?:\.\d+)$", sent_at):
```

Reply:
Oops, of course